### DISCUSSION TITLE
GIF can not work when it is exported to PDF and pptx

### DISCUSSION THREAD

**Discussion by LouisYZK** [marp-team/marp#223] [Q&A] [answered]
When i export my slides with .gif figure to .pptx and .pdf, the gif cannot work, it became static like a jpg

**Comment by yhatt** [comment, marked as answer]
Works as designed. See following issues for details:

- https://github.com/marp-team/marp-cli/issues/196
- https://github.com/marp-team/marp-cli/issues/249
